website/integrations: add OVH (#14664)
goauthentik/authentik: website/integrations/services/ovh/index.md

---

> title: Integrate with OVH Cloud

Reviewer: ```suggestion
title: Integrate with OVHcloud
```

> sidebar_label: OVH Cloud

Reviewer: ```suggestion
sidebar_label: OVHcloud
```

support_level: community
---

> ## What is OVH Cloud

Reviewer: ```suggestion
## What is OVHcloud
```

> > OVH Cloud is a French cloud provider. They provide public and private cloud products, shared hosting, and dedicated servers in 140 countries.

Reviewer: ```suggestion
> OVHcloud is a French cloud provider. They provide public and private cloud products, shared hosting, and dedicated servers in 140 countries.
```

>
> -- https://www.ovhcloud.com

## Preparation

The following placeholders are used in this guide:

- `0123-456-78` is your OVH Customer Code.
- `authentik.company` is the FQDN of the authentik installation.

## authentik Configuration

> To support the integration of OVH Cloud with authentik, you need to create an application/provider pair in authentik.

Reviewer: ```suggestion
To support the integration of OVHcloud with authentik, you need to create an application/provider pair in authentik.
```

    - **Application**: Provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
    - **Choose a Provider type**: Select **SAML Provider** as the provider type.
    - **Configure the Provider**: Provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations:
        - Set the **ACS URL** to `https://www.ovhcloud.com/eu/auth/saml/acs`.
        - Set the **Service Provider Binding** to `Post`.
        - In **Advanced protocol settings**, select a **Signing Certificate** and check that **Sign assertions** is enabled.

    - **Configure Bindings** _(optional)_: You can create a [binding](/docs/add-secure-apps/flows-stages/bindings/) (policy, group, or user) to manage the listing and access to applications on a user's **My applications** page.

3. Click **Submit** to save the new application and provider.

> 

Reviewer: ```suggestion

### Download metadata file

1. Log in to authentik as an admin, and open the authentik Admin interface.
2. Navigate to **Applications** > **Providers** and click on the name of the provider that you created in the previous section (e.g. `Provider for OVHcloud`).
3. Under **Related objects** > **Metadata**, click on **Download**. This downloaded file is your `SAML Metadata` file and it will be required in the next section.

```

> ## Service Configuration

Reviewer: ```suggestion
## OVHcloud Configuration
```

> 1. Go to [OVH Cloud](https://www.ovhcloud.com) and log in.

Reviewer: ```suggestion
1. Log in to the [OVHcloud Control Panel](https://www.ovh.com/auth/).
```

2. Click on your **name** in the top right corner. Then, click on your **name** again in the sidebar that opens.
3. Select **Identity and Access Management (IAM)** from the left-hand menu.
4. Click the **Identities** tab to access local users management and switch to the **SSO** tab.
5. Click on the **SSO Connection** button.
    - If you want to keep local OVH users, tick the `Keep active OVHcloud users` box.

>     - Copy the XML-Metadata from your authentik installation (Applications -> Providers -> Your OVH Provider -> Metadata) to the **XML-Metadata** field.

Reviewer: This will need to be updated to explain in more detail how to paste in the metadata information.

Author: I tried. Hopefully, this description is more detailed, but I cannot find a phrasing I'm really happy with.
